addons/lunch/static/src/js/lunch_kanban_widget.js

```diff
@@ -73,6 +58,25 @@ var LunchKanbanWidget = Widget.extend({
 
         this.currency = params.currency || session.get_currency(session.company_currency_id);
     },
+    willStart: function () {
```

Comment:
a fix without a test is not a fix, it is either useless code or a bug in waiting ;)

Reply:
I do agree about the fix/test relationship and I will write one (should have been done actually, my bad)... 

But wouldn't it be weird afterward to see a dont-use-the-framework-wrongly test? Shouldn't tests describe what the system do? Will it make sense on a test-suite point of view?

Reply:
PS: this is not a troll :stuck_out_tongue_winking_eye: